uniffi_bindgen/src/interface/attributes.rs

```diff
@@ -23,12 +23,13 @@ use anyhow::{bail, Result};
 /// This is a convenience enum for parsing UDL attributes and erroring out if we encounter
 /// any unsupported ones. These don't convert directly into parts of a `ComponentInterface`, but
 /// may influence the properties of things like functions and arguments.
-#[derive(Debug, Clone, Hash)]
+#[derive(Debug, Clone, Hash, PartialEq)]
 pub(super) enum Attribute {
     ByRef,
     Error,
     Threadsafe,
     Throws(String),
+    SomethingSomethingArc,
```

Comment:
I don't really understand the need for this, so I'm not going to much help naming this.